pacientes/models.py

```diff
@@ -194,35 +195,34 @@ def create_from_sss(cls, dni):
-                    oss, created = ObraSocial.objects.get_or_create(
-                        codigo=oss_codigo,
-                        defaults={
-                            'nombre': oss_data['Denominación Obra Social'],
+                oss_data = tablas[1]['data']
+                oss_codigo = (''.join(filter(str.isdigit, oss_data['Código de Obra Social'])))
+                oss, created = ObraSocial.objects.get_or_create(
+                    codigo=oss_codigo,
+                    defaults={
+                        'nombre': oss_data['Denominación Obra Social'],
+                        })
+                ObraSocialPaciente.objects.create(
+                    data_source=settings.SOURCE_OSS_SSS,
+                    paciente=paciente,
+                    obra_social_updated=now(),
+                    obra_social=oss,
+                    tipo_beneficiario=data['Parentesco'].lower()
+                )
+                if tam_tabla >= 3:
+                    empleador_data = tablas[2]['data']
+                    cuit_parseado = (''.join(filter(str.isdigit, oss_data['CUIT de empleador'])))
+                    empresa, created = Empresa.objects.get_or_create(cuit=cuit_parseado, defaults={
+                                'nombre': empleador_data['Tipo Beneficiario Declarado'],
                             })
-                    ObraSocialPaciente.objects.create(
-                        data_source=settings.SOURCE_OSS_SSS,
+                    fecha = empleador_data['Ultimo Período Declarado'].split('-')
+                    EmpresaPaciente.objects.get_or_create(
                         paciente=paciente,
-                        obra_social_updated=now(),
-                        obra_social=oss,
-                        tipo_beneficiario=data['Parentesco'].lower()
+                        empresa=empresa,
+                        ultimo_recibo_de_sueldo=datetime.datetime(int(fecha[1]), int(fecha[0]), 1)
                     )
-                    if tam_tabla >= 3:
-                        empleador_data = tablas[2]['data']
-                        cuit_parseado = (''.join(filter(str.isdigit, oss_data['CUIT de empleador'])))
-                        empresa, created = Empresa.objects.get_or_create(cuit=cuit_parseado, defaults={
-                                    'nombre': empleador_data['Tipo Beneficiario Declarado'],
-                                })
-                        fecha = empleador_data['Ultimo Período Declarado'].split('-')
-                        EmpresaPaciente.objects.get_or_create(
-                            paciente=paciente,
-                            empresa=empresa,
-                            ultimo_recibo_de_sueldo=datetime.datetime(int(fecha[1]), int(fecha[0]), 1)
-                        )
                 return True, paciente
-            elif tablas[0]['name'] == 'NO_AFILIADO':
+            elif not res['resultados']['afiliado']:
```

Comment:
Acá creo que sería lo mismo poner un else.

```suggestion
            else:
```